spacy/gold.pxd

```diff
@@ -36,6 +36,19 @@ cdef class GoldParse:
 
     cdef readonly list cand_to_gold
     cdef readonly list gold_to_cand
-    cdef readonly list orig_annot
 
 
+cdef class RawAnnot:
```

Comment:
We could just make this a normal class? We would lose the compile-time checking, but gain a bit of flexibility.